src/features/workflow/state/workflowState.ts

```diff
@@ -42,6 +44,13 @@ export const selectLatestDryRunId = (state: RootState): string => {
   return ''
 }
 
+export const selectLatestDryRunDeployId = (state: RootState): string => {
```

Comment:
should modify this function name to match it's priority, how about `selectLatestDeployOrDryRunId`?